Dockerfile

```diff
@@ -37,7 +41,7 @@ USER runner:runner
 ARG builddate="(unknown)"
 ARG version="(unknown)"
 LABEL org.label-schema.build-date="${builddate}"
-LABEL org.label-schema.description="ocs-monkey workload simulator"
+LABEL org.label-schema.description="ocs-monkey chaos runner"
```

Comment:
This container image is now being used for both the workload and the chaos runner, correct?